### ISSUE TITLE
Exported object files are empty .. except for one

### ISSUE THREAD

**Issue by daveespo** [dcarroll/sfdx-djc-plugin#3]
I have a hierarchy of objects in my Developer Edition org (I'm not using a scratch org right now)

Starting simply, the schema is:

````
TicketableEvent__c
  EventInstance__c
    TicketAllocation__c
      TicketPriceLevel__c
````

I ran the following command

````
sfdx djc:data:export -o "EventInstance__c,TicketableEvent__c,TicketAllocation__c,TicketPriceLevel__c" -u sfptdev.wolverine.dave -t export2
````

I'm attaching the content of the export2 directory -- don't worry, this is all bogus test data

Note that only TicketableEvent__c is populated with records. The other 3 child/grandchild/great-grandchildren are empty

Also note that TicketableEvent__c.json only has about 10 fields for each record -- the SOBject has dozens of fields.

I'm running the live copy of this plugin (sfdx plugins:link) right off master

How can I help further debug this?

**Comment by dcarroll** [comment]
Sorry for the delay, just returned from holiday. Thanks for reporting this. Can you share the metadata for the Sobjects?  This wold be helpful for me to debug the issue.

**Comment by dcarroll** [comment]
I've tried to duplicate the issue and could not.  I used the metadata and data at https://github.com/dcarroll/dreamhouse-with-tickets to test.

**Comment by daveespo** [comment]
[objects.zip](https://github.com/dcarroll/sfdx-djc-plugin/files/2503878/objects.zip)

Attached is the MDAPI format metadata for those 4 objects.

They contain Lookup references to other custom SObjects in the same (and in some cases) other namespaces so if you need those, I can provide a more complete schema -- I'd prefer to send those over via email vs. posting them on GH

Also, I can grant login access to this org if you care to poke and prod it directly

Thanks!

**Comment by dcarroll** [comment]
Hey @daveespo .  I've just done a major update that should take care of this.  Although, I will caveat that I am not sure what namespacing will do to the process.  My email address is easily discoverable as is my twitter handle if you want a more private forum to work through this.

**Comment by daveespo** [comment]
Yes, great progress forward here. I now have all of the files populated but I am indeed running afoul of problems including locally defined custom fields on managed package custom objects. (i.e. if pkg1 defines PaymentTransaction__c and my local project adds a custom fields to that object, it's not included in the export files)

Similarly, Lookups to Standard objects (Account/Contact) are not included in the dumped fields.

I'll send you email with the whole MDAPI project
